hphp/ppc64-asm/decoded-instr-ppc64.cpp

```diff
@@ -438,6 +468,13 @@ bool DecodedInstruction::isLimmediatePossible() const {
   return isLi64Possible();
 }
 
+bool DecodedInstruction::isLoadingTOC() const {
+  if (m_dinfo.isLd(true) || m_dinfo.isLwz(true) || m_dinfo.isAddis(true)) {
+    return true;
+  }
+  return false;
```

Comment:
`return m_dinfo.isLd(true) || ...;`

You don't need to check a boolean for true; just return it.

Reply:
Done.